### PR TITLE
Networks improvements

### PR DESCRIPTION
**Pull request by mkurek**
- default margin (top and bottom) set to 10
- status field added to IP form
- IP baseobject limited to [DataCenterAsset, Database, VIP, VirtualServer, CloudHost]
- added list of networks in which object (DC Asset) is a terminator
- "pretty name" of linked object in IP Address list view
- other small fixes & improvements (queries count, additional fields etc)

### CODE REVIEW

File: src/ralph/settings/base.py [hunk 1]
```diff
@@ -283,3 +283,6 @@ def os_env_true(var, default=''):
 
 # issue tracker url for Operations urls (issues ids) - should end with /
 ISSUE_TRACKER_URL = os.environ.get('ISSUE_TRACKER_URL', '')
+
+# Networks
+DEFAULT_NETWORK_MARGIN = 10
```
Comment (ghost) [inline src/ralph/settings/base.py]: `os.environ`? :)

**Comment by ghost** [comment]
:ok:

**Comment by ar4s** [comment]
:ok:

**Comment by szok** [comment]
:ok:

**Pull request merged**